Sources/GRPC/GRPCStatus.swift

```diff
@@ -75,19 +70,25 @@ extension GRPCStatus {
   /// Status codes for gRPC operations (replicated from `status_code_enum.h` in the
   /// [gRPC core library](https://github.com/grpc/grpc)).
   public struct Code: Hashable, CustomStringConvertible {
-    public let rawValue: Int
+    // `rawValue` must be an `Int` for API reasons and we don't need (or want) to store anything so
+    // wide, a `UInt8` is fine.
+    private let _rawValue: UInt8
+
+    public var rawValue: Int {
+      return Int(self._rawValue)
+    }
 
     public init?(rawValue: Int) {
       switch rawValue {
       case 0 ... 16:
-        self.rawValue = rawValue
+        self._rawValue = UInt8(rawValue)
```

Comment:
I mean, this is a really silly performance optimisation that we totally don't need, but it's nice to have: given that we have the check above, we know the value is in-bounds for UInt8. So we can use `UInt8(truncatingIfNeeded: rawValue)` and save ourselves some branches.

Reply:
https://swift.godbolt.org/z/jG1YYv

Reply:
Silly, but no point leaving it on the table. Plus, I think it's generally useful to be thinking about these optimisations.